db/migrate/20200802210346_create_agencies.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAgencies < ActiveRecord::Migration[6.0]
+  def change
+    create_table :agencies, id: :uuid do |t|
+      t.string :name, null: false
+      t.string :state
+      t.boolean :active, null: false, default: true
+
+      t.timestamps
+    end
+  end
```

Comment:
Can we add a compound unique index here so no one can add an agency twice?

Something like
```
add_index :agencies, [:name, :state], unique: true
```